pkg/diagnostics/cluster/roles.go

```diff
@@ -20,7 +20,34 @@ type ClusterRoles struct {
 }
 
 const (
-	ClusterRolesName = "ClusterRoles"
+	ClusterRolesName   = "ClusterRoles"
+	clusterRoleMissing = `
+clusterrole/%s is missing.
+
+Use the 'oadm policy reconcile-cluster-roles' command to create the role. For example,
+
+  $ oadm policy reconcile-cluster-roles \
```

Comment:
> I'm not sure if it's the case here,

Most probably not the case... these seems to be just constants that I believe are printed as output of the diagnostics.